src/read/mmcif/parser.rs

```diff
@@ -610,6 +624,102 @@ fn parse_atoms(input: &Loop, pdb: &mut PDB, options: &ReadOptions) -> Option<Vec
     }
 }
 
+/// Parse a loop containing secondary structure data.
+fn parse_helices(input: &Loop, pdb: &mut PDB, options: &ReadOptions) -> Option<Vec<PDBError>> {
+    // todo: DRY with parse_atoms throughout this function.
+
+    define_columns!(
```

Comment:
It might be nice to check the mmCIF docs for which items are required: https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Categories/struct_conf.html.